### PR TITLE
1/2 Feat: Scanner

### PR DESCRIPTION
**Pull request by Setanna**

### CODE REVIEW

**Comment by Setanna** [comment]
[GUI - Scanner](https://trello.com/c/qZTi2rXV/26-gui-scanner)

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -74,10 +85,21 @@ public function edit(string $id)
      * @param Product $product
      * @return ProductResource
      */
-    public function update(ProductRequest $request, Product $product): ProductResource
+    public function update(ProductRequest $request, Product $product) //: ProductResource
```
Comment (MathiasDuus) [inline app/Http/Controllers/ProductController.php]: Should it be commented

**Review by MathiasDuus — CHANGES_REQUESTED** [review]
One small change else looks good

File: app/Http/Controllers/ProductController.php (hunk 1 above)
Comment (Setanna) [reply app/Http/Controllers/ProductController.php]: Uncommented it to again correctly apply return type

**Review by MathiasDuus — APPROVED** [review]

**Comment by Setanna** [comment]
[GUI - Scanner](https://trello.com/c/qZTi2rXV/26-gui-scanner)

**Pull request merged**